tests/transaction/test_get_list_transactions.py

```diff
@@ -0,0 +1,286 @@
+"""
+Provide tests for command line interface's get list of transactions command.
+"""
+import json
+
+import pytest
+from click.testing import CliRunner
+
+from cli.constants import (
+    FAILED_EXIT_FROM_COMMAND_CODE,
+    NODE_IP_ADDRESS_FOR_TESTING,
+    PASSED_EXIT_FROM_COMMAND_CODE,
+)
+from cli.entrypoint import cli
+from cli.utils import dict_to_pretty_json
+
+
+def test_get_list_transaction_with_ids():
```

Comment:
```suggestion
def test_get_list_transactions_with_ids():
```
Change it in all test cases below, please.

Reply:
@anastasiia-bilova, done.